### PR TITLE
Remove dev repositories and Bintray usages where applicable

### PR DESCRIPTION
**Pull request by qwwdfsad**

### CODE REVIEW

**Comment by qwwdfsad** [comment]
Filed #2590

File: js/example-frontend-js/build.gradle.kts [hunk 1]
```diff
@@ -21,7 +21,13 @@ kotlin {
     }
 }
 
+// For kotlinx-html
+repositories {
+    maven("https://kotlin.bintray.com/kotlinx")
```
Comment (elizarov) [inline js/example-frontend-js/build.gradle.kts:26]: It has moved to space `maven("https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven")`

File: js/example-frontend-js/build.gradle.kts [hunk 2]
```diff
@@ -21,7 +21,13 @@ kotlin {
...
+}
+
 dependencies {
+
```
Comment (elizarov) [inline js/example-frontend-js/build.gradle.kts:30]: Nit: extra line

File: build.gradle [hunk 3]
```diff
@@ -53,16 +53,8 @@ buildscript {
...
-        maven { url "https://cache-redirector.jetbrains.com/maven.pkg.jetbrains.space/kotlin/p/kotlin/dev" }
-        maven {
-            url "https://kotlin.bintray.com/kotlin-dev"
-            credentials {
-                username = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER') ?: ""
-                password = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_API_KEY') ?: ""
-            }
-        }
+        // Leftover until we migrated to Dokka 1.4.30
+        maven { url "https://kotlin.bintray.com/kotlin-dev" }
```
Comment (elizarov) [inline build.gradle:57]: It looks like this one should also conditionally use a cache-redirected URL (`"https://cache-redirector.jetbrains.com/maven.pkg.jetbrains.space/kotlin/p/kotlin/dev"`) to be fully bintray-independent on our CI.

**Review by elizarov — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 3 above)
Comment (qwwdfsad) [reply build.gradle:57]: It's already being conditionally redirected, isn't it? https://github.com/Kotlin/kotlinx.coroutines/blob/master/build.gradle#L82

Comment (elizarov) [reply build.gradle:57]: Thanks. Missed it!

**Review by elizarov — APPROVED** [review]
Marvelous!

**Pull request merged**